spec/integration/change_stream_examples_spec.rb

```diff
@@ -174,10 +174,12 @@
 
         # Start Changestream Example 4
 
-        pipeline = [ {'$match' => { '$or' => [{ 'fullDocument.username' => 'alice' },
-                                              { 'operationType' => 'delete' }] } }]
-        cursor = inventory.watch(pipeline).to_enum
-        cursor.next
+        pipeline = [
+          { "$match" => { 'fullDocument.username' => 'alice' } },
+          { "$addFields" => { 'newField' => 'this is an added field!' } }
+        ];
+        changeStream = inventory.watch(pipeline).to_enum
+        changeStream.next
 
         # End Changestream Example 4
```

Comment:
After this line please add an assertion that the document retrieved from the change stream has the expected fields (i.e. the `newField` is present). You can look at other examples to see how they assert after the example itself.

Reply:
Done!